apps/asap-server/src/controllers/users.ts

```diff
@@ -226,6 +227,41 @@ export default class Users implements UserController {
     return parseGraphQLUser(items[0]);
   }
 
+  async fetchByLabId(
```

Comment:
@kmaid @marot what do you think?

Reply:
I think having one endpoint that can handle complex queries would be better. We might be able to reuse different combinations of filters etc to solve future requirements rather than having to create a brand new endpoint

Reply:
I also think we really should have documentation for our endpoints and what filters they take

Reply:
It looks like `fetchByLabId` takes the `FetchOptions` parameter but only uses it partially. I think it makes sense
to integrate this lab filter into the fetch function

Reply:
yeah totally agree (which might be obvious since i was the one to suggest it) but with one remark - you guys do realise that  controllers != endpoints though right? What i think we want to document are http endpoints since this is the interface the API exposes. But yeah eventually this comes down to using whatever the controllers offer